app/src/main/java/com/android/sample/ui/activity/EditActivity.kt

```diff
@@ -343,9 +344,11 @@ fun EditActivityScreen(
                   } catch (_: Exception) {}
                 }
 
-                Toast.makeText(
-                        context, "Invalid format, date must be DD/MM/YYYY.", Toast.LENGTH_SHORT)
-                    .show()
+                if (parts.size != 3) {
+                  Toast.makeText(
+                          context, "Invalid format, date must be DD/MM/YYYY.", Toast.LENGTH_SHORT)
+                      .show()
+                }
```

Comment:
The addition of more descriptive toasts for input errors helps guide users more clearly, reducing user frustration and potential input mistakes.